### PR TITLE
feat: apply filter, video listing

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:

#Description: Added list of Videos, Filter by category

## More details:
1. User can see videos on the landing page
2. Apply filter to see videos of a particular category
3. Used context

What not to check?
- Responsiveness
- Using css variables

Screenshots:
av2
av3

### CODE REVIEW

File: src/components/VideoCard/VideoCard.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import "./VideoCard.css";
+
+export default function VideoCard({ video, selectOptions }) {
+  const { _id, title, creator, date, views, photo } = video;
+  const [showOptions, setShowOptions] = useState(true);
+
+  return (
+    <div className="video-card">
+      <Link className="video-link" to={`/watchvideo/${_id}`}>
+        {/* <iframe src={url} title={title} allowFullScreen></iframe> */}
```
Comment (tushargoyal812) [inline src/components/VideoCard/VideoCard.js:12]: you can remove comments

File: src/context/filter-context.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+//     case "pop":
+//       return "pop";
+//     case "EDM":
+//       return "EDM";
+//     case "Hip Hop":
+//       return "Hip Hop";
+//     default:
+//       return "";
+//   }
+// };
```
Comment (tushargoyal812) [inline src/context/filter-context.js:32]: remove commented code if not required

**Review by tushargoyal812 — COMMENTED** [review]
design looks good.
code is clean and to the point,just remove comments.
variable naming is good

**Review by TejasShekar — APPROVED** [review]
Good file structure.
Liked the UI.
Looks good to merge.
One suggestion would be to user good classNames.

File: src/context/filter-context.js (hunk 2 above)
Comment (krgarima) [reply src/context/filter-context.js:32]: Thank you for telling.

**Pull request merged**